src/cli/onefuzz/debug.py

```diff
@@ -169,6 +170,19 @@ def _get_node(
 
         raise Exception("unable to find scaleset node running on task")
 
+    def disable_node_reset(
+        self, task_id: UUID_EXPANSION, *, node_id: Optional[UUID] = None
+    ) -> None:
+        while True:
+            try:
+                scaleset_id, node_id = self._get_node(task_id=task_id, node_id=node_id)
```

Comment:
How about moving those two operation on the server side to minimize the the chances that a node stops running the task before we update it

Reply:
Let's talk about this offline.  I wasn't happy with any of the endpoint APIs I prototyped to expose this.